libcodechecker/libclient/client.py

```diff
@@ -29,7 +32,20 @@ def check_api_version(client):
     """
 
     version = client.getAPIVersion()
-    supp_major_version = SUPPORTED_VERSION.split('.')[0]
+    supp_major_version = SUPPORTED_RUN_API_VERSION.split('.')[0]
+    api_major_version = version.split('.')[0]
+
+    # There is NO compatibility between major versions.
+    return supp_major_version == api_major_version
+
+
+def check_product_api_version(client):
+    """
+    Check if server product API is supported by the client.
+    """
+
+    version = client.getAPIVersion()
```

Comment:
It is not the same version. There is a database version, a product API version, and a run API version. The fact that they all are `6.0` right now is merely coincidental, as we are heavily breaking API and not versioning it at this point, but there will be one release which will set forth versioning from `6.0` and onwards.

Same as how the database migration is not handled, as the whole infrastructure in a big WIP.